packages/ember-data/lib/system/model/model.js

```diff
@@ -33,6 +33,11 @@ var Model = Ember.Object.extend(Ember.Evented, {
 
   store: null,
 
+  // Remove support for `record.container`
+  get container() {
+    Ember.assert("A model's private reference to the container has been removed for consistency reasons. Instances of a store (i.e. `this.store`) have access to the container, although usage from within a model is discouraged and can lead to unintended consequences.");
+  },
+
```

Comment:
@igorT I read that beta.19 is the last ED release to support IE8 :trollface: 

Seriously though, I started by doing a `Ember.defineProperty` on each model instance right after it's created, but it seemed like a bad choice for perf.

Everything passes when running tests in the browser, but for some reason destroys all tests in node. Debugging in node is proving to be fruitless, is this something I should even be trying to do?